collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -0,0 +1,79 @@
+package org.odk.collect.android.preferences;
+
+import static org.odk.collect.android.preferences.AdminAndGeneralKeys.ag;
+
+public final class AdminKeys {
+    // key for this preference screen
+    public static String KEY_ADMIN_PW = "admin_pw";
+
+    // keys for each preference
+
+    // main menu
+    public  static String KEY_EDIT_SAVED                = "edit_saved";
```

Comment:
Good observation. I missed that myself. They were that way when I moved them to the new file, but I missed an opportunity to improve them. Want to take any action now?